pkg/servicequota/vpcusage.go

```diff
@@ -0,0 +1,71 @@
+package servicequota
+
+import (
+	"fmt"
+
+	"github.com/oslokommune/okctl/pkg/config"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/servicequotas"
+	"github.com/logrusorgru/aurora"
+	"github.com/oslokommune/okctl/pkg/api/okctl.io/v1alpha1"
+)
+
+type vpcusage struct {
+	CloudProvider v1alpha1.CloudProvider
+}
+
+func (v vpcusage) Count() (int, error) {
```

Comment:
Prefer to operate on a pointer to the struct, otherwise, we are doing a lot of copy by value, which is good when we want to protect the internal data/state, but that is not the case for these checks.

Reply:
Done :)